packages/kit/test/apps/basics/src/routes/redirect-on-load/_tests.js

```diff
@@ -0,0 +1,9 @@
+import * as assert from 'uvu/assert';
+
+/** @type {import('test').TestMaker} */
+export default function (test, is_dev) {
+	test('redirect-on-load', '/redirect-on-load', async ({ base, page, clicknav }) => {
```

Comment:
I don't think this is testing the new functionality you added. I think it's testing server-side redirect. If you look at some of the other tests there should be examples of testing both

Reply:
Oops. sorry!
I updated!
(This time, again I checked tests that fails without this PR but passes with it.)

And during the update, I faced puppeteer issue due to redirect.
So I edited `packages/kit/test/test.js` in the last commit.